internal/cli/opsmanager/metrics/process.go

```diff
@@ -38,14 +38,19 @@ func (opts *ProcessOpts) initStore() error {
 	return err
 }
 
+var metricTemplate = `NAME	UNITS			TIMESTAMP		VALUE{{range .Measurements}}
+{{.Name}}	{{.Units}}{{range .DataPoints}}	
+			{{.Timestamp}}	{{.Value}}{{end}}{{end}}
+`
```

Comment:
Similar comment to above, to make this more readable we should have a name and unit along the values